content/docs/1.3.0/snapshots-and-backups/csi-snapshot-support/_index.md

```diff
@@ -1,11 +1,30 @@
 ---
 title: CSI Snapshot Support
-description: Creating and Restoring Longhorn Backups via the kubernetes CSI snapshot mechanism
+description: Creating and Restoring Longhorn Snapshots/Backups via the kubernetes CSI VolumeSnapshot mechanism
 weight: 3
 ---
 
 ## History
-- [GitHub Issue](https://github.com/longhorn/longhorn/issues/304)
-- [Longhorn Enhancement Proposal](https://github.com/longhorn/longhorn/blob/master/enhancements/20200904-csi-snapshot-support.md)
+- GitHub Issues:
+  - https://github.com/longhorn/longhorn/issues/304
+  - https://github.com/longhorn/longhorn/issues/2534
+- Longhorn Enhancement Proposals:
+  - https://github.com/longhorn/longhorn/blob/master/enhancements/20200904-csi-snapshot-support.md
+  - https://github.com/longhorn/longhorn/blob/master/enhancements/20220110-extend-csi-snapshot-to-support-longhorn-snapshot.md
+- Availability:
+  - CSI VolumeSnapshot associated with Longhorn backup is available since v1.1.0
+  - CSI VolumeSnapshot associated with Longhorn snapshot is available since v1.3.0
 
-Available since v1.1.0
+## Overview
+
+In Kubernetes, a [CSI VolumeSnapshot](https://kubernetes.io/docs/concepts/storage/volume-snapshots/) represents a snapshot of a volume on a storage system.
+When you create a CSI VolumeSnapshot, the storage system will take a snapshot of the corresponding volume.
+You can then use the CSI VolumeSnapshot to create a new volume that has the same content as the content of the CSI VolumeSnapshot.
+
+In Longhorn, there are 2 concepts related to the content of a volume at particular moment: [Longhorn backup](../../concepts/#3-backups-and-secondary-storage) and [Longhorn snapshot](../../concepts/#24-snapshots).
```

Comment:
Suggest describes snapshot first then backup.

```suggestion
In Longhorn, there are 2 concepts related to the content of a volume at a particular moment: [Longhorn snapshot](../../concepts/#24-snapshots) and [Longhorn backup](../../concepts/#3-backups-and-secondary-storage).
```

Reply:
Thank you. Fixed